backend/manager/modules/dal/src/main/java/org/ovirt/engine/core/dao/VmBackupDaoImpl.java

```diff
@@ -40,7 +40,8 @@ protected MapSqlParameterSource createFullParametersMapper(VmBackup entity) {
                 .addValue("_update_date", entity.getModificationDate())
                 .addValue("description", entity.getDescription())
                 .addValue("backup_type", entity.getBackupType().getName())
-                .addValue("snapshot_id", entity.getSnapshotId());
+                .addValue("snapshot_id", entity.getSnapshotId())
+                .addValue("is_stopped", entity.isStopped());
```

Comment:
if we go with my proposal above then we don't need to add it here - the backup is created with is_stopped=FALSE and only stop-backup would be able to change it (so it won't be overridden by this call)